quilt/test/test_build.py

```diff
@@ -96,3 +97,49 @@ def test_failover(self):
         path = os.path.join(mydir, './build_failover.yml')
         build.build_package('test_failover', PACKAGE, path)
         from quilt.data.test_failover.groot import bad
+
+    def test_duplicates(self):
```

Comment:
This test is failing for me in Python 3.5.3:

E       AssertionError: assert {'a_b_c': {'f...xt.csv'}, ...} == {'a': {}, 'a_b...'a.txt'}, ...}
E         Common items:
E         {'a_b_c': {'file': 'a%%b___c'},
E          'a_csv': {'file': 'a.csv'},
E          'a_txt': {'file': 'a.txt'},
E          'a_txt_csv': {'file': 'a_txt.csv'},
E          'n1': {'file': '1'},
E          'n123': {'file': '_123'},
E          'n1_txt': {'file': '1.txt'}}
E         Right contains more items:
E         {'a': {}, 'dir_ext': {}}
E         Full diff:
E         + {'a': {},
E         - {'a_b_c': {'file': 'a%%b___c'},
E         ? ^
E         +  'a_b_c': {'file': 'a%%b___c'},
E         ? ^
E         'a_csv': {'file': 'a.csv'},
E         'a_txt': {'file': 'a.txt'},
E         'a_txt_csv': {'file': 'a_txt.csv'},
E         +  'dir_ext': {},
E         'n1': {'file': '1'},
E         'n123': {'file': '_123'},
E         'n1_txt': {'file': '1.txt'}}